### PR TITLE
Support python24 e.g. for RHEL/CentOS5

### PR DESCRIPTION
**Pull request by tyll**

### CODE REVIEW

File: library/zone_serial [hunk 1]
```diff
@@ -146,17 +146,19 @@ def main():
 
     # Fetch the zone file with serial to fill in:
     try:
-        with open(src, 'r') as fh:
-            src_data = fh.read()
-    except IOError as e:
+        fh = open(src, 'r')
```
Comment (lurtz-brainly) [inline library/zone_serial]: If you hit IOException, handler for file will be not closed. Please fix it.

**Comment by 23inhouse** [comment]
This has been open since May 13th. Should it be closed? /cc @tyll, @nogeek

**Comment by 23inhouse** [comment]
I'm going to go ahead and close this one. Please re-open it with the suggested fixed at any time.

**Comment by nogeek** [comment]
:+1:

**Pull request closed without merging**